doc/releases/changelog-dev.md

```diff
@@ -413,6 +413,12 @@
 
 <h3>Breaking changes 💔</h3>
 
+* Decompositions of two-qubit unitaries via `two_qubit_decomposition` now use 3 CNOTs even if
+  just 2 CNOTs would suffice theoretically. This may or may not affect a user's workflow, depending
+  on the numerical matrices provided to the decomposition. This change fixes a previous bug that
+  affected the correctness of the decomposition in some edge cases.
```

Comment:
I'm still a little confused here 😵. If 2 CNOTs suffice, then why was there a bug? Perhaps this is a better wording (less words 😅).

```suggestion
  just 2 CNOTs would suffice theoretically. This fixes a bug in the implementation for this decomposition
  that affected its correctness in some edge cases.
```

Reply:
scratch that, given your other feedback I rewrote it.